htdocs/composer.json

```diff
@@ -0,0 +1,85 @@
+{
+	"name": "impresscms/impresscms",
+	"description": "ImpressCMS - A dynamic and user-friendly Content Management System",
+	"type": "project",
+	"keywords": ["cms", "content-management", "php", "mysql", "web"],
+	"homepage": "https://www.impresscms.org/",
+	"license": "GPL-2.0-or-later",
+	"authors": [
+		{
+			"name": "The ImpressCMS Project",
+			"homepage": "https://www.impresscms.org/",
+			"role": "Developer"
+		}
+	],
+	"require": {
+		"php": ">=7.4.0",
```

Comment:
I started with PHP 7.4 and after running composer update in my docroot, I got this when I tried to start the install - 

> Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0".